spring-native-configuration/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/MetricsEndpointAutoConfigurationHints.java

```diff
@@ -19,18 +19,16 @@
 import org.springframework.boot.actuate.autoconfigure.metrics.export.properties.PropertiesConfigAdapter;
 import org.springframework.boot.actuate.autoconfigure.metrics.export.simple.SimplePropertiesConfigAdapter;
 import org.springframework.boot.actuate.metrics.MetricsEndpoint;
-import org.springframework.nativex.hint.TypeAccess;
 import org.springframework.nativex.hint.NativeHint;
+import org.springframework.nativex.hint.TypeAccess;
 import org.springframework.nativex.hint.TypeHint;
 import org.springframework.nativex.type.NativeConfiguration;
 
 // Hitting /metrics endpoint - needs tests
 @NativeHint(trigger = MetricsEndpointAutoConfiguration.class, types = {
 	@TypeHint(types = {
 		PropertiesConfigAdapter.class,
-		SimplePropertiesConfigAdapter.class,
-		org.springframework.boot.actuate.autoconfigure.metrics.ServiceLevelObjectiveBoundary.class,
-		org.springframework.boot.actuate.autoconfigure.metrics.ServiceLevelObjectiveBoundary[].class,
```

Comment:
Since `ServiceLevelObjectiveBoundary.class` and `ServiceLevelObjectiveBoundary[].class` require distinct configuration (at least it was the case until now), should we keep `ServiceLevelObjectiveBoundary[].class` here since it was needed for the endpoint as far as I remember?

Reply:
The `ServiceLevelObjectiveBoundary[]` entries are still in the generated `reflect-config.json` file:

```
  {
    "name": "org.springframework.boot.actuate.autoconfigure.metrics.ServiceLevelObjectiveBoundary"
  },
  {
    "name": "org.springframework.boot.actuate.autoconfigure.metrics.ServiceLevelObjectiveBoundary[]"
  }
```

I'm not sure where the array one is coming from. But of course we can explicitly include them, just to be sure.

Reply:
Ok likely generated by the improved inference mechanism I guess.